src/components/api/api.ts

```diff
@@ -162,6 +162,8 @@ export const fetchPriceAndStock = async (ingredientIds: {
         textualAmount: products[index].textualAmount,
         badges: products[index].badges,
         image: products[index].images[0],
+        tooltips: stock[index].tooltips,
```

Comment:
Premyslim jestli si to tady nejak nenormalizovat, treba vzit jen ten prvni toolip? Muzem poresit na callu. 🙏 

Protoze se zda, ze stejne pracujes jen s `tooltips[0].xyz`